src/components/tree/tree.stories.ts

```diff
@@ -64,11 +115,17 @@ export const simple = (): string => html`
   </calcite-tree>
 `;
 
-export const selectionModeNone = (): string => html`<calcite-tree
-  ${boolean("lines", false)}
-  selection-mode="${select("selection-mode", selectionModes, "none")}"
+export const selectionModeNone = (): string => html`
+  <calcite-tree ${boolean("lines", false)} selection-mode="${select("selection-mode", selectionModes, "none")}"
+    >${treeItems}</calcite-tree
+  >
+`;
+
+export const actionsEndDropdownsAndIconStart = (): string => html`<calcite-tree
```

Comment:
This looks like a test-specific story. If so, can you add the `_TestOnly` suffix?